### DISCUSSION TITLE
vscode-graphql-execution not running

### DISCUSSION THREAD

**Discussion by PixeledCode** [graphql/graphiql#3170] [Q&A / Support]
Attaching the screenshot. I have the latest extension installed, noticed I was not getting any codelens in `.graphql` files. Language Feature and Syntax Highlighting are working as expected.
I also do not see any log for `vscode-graphql-execution`. Is that normal?

<img width="1359" alt="Screenshot 2023-05-10 at 2 25 37 PM" src="https://github.com/graphql/graphiql/assets/22215209/fbc23bd5-b96d-447e-b066-f5b49c635482">

**Comment by acao** [comment]
it appears there are still some issues we need to work out with this extension, thanks for letting me know! it has something to do with the bundling, because it works locally.

**Reply by PixeledCode** [reply]
thanks for the update :)

**Comment by gastonmorixe** [comment]
related?  opened vscode from terminal and log debug mode `code --log debug .` 

<img width="1250" alt="image" src="https://github.com/graphql/graphiql/assets/637225/d1129417-c5d1-4a89-b644-2b0ed9746d21">
